plugins/rule_engines/irods_rule_language/src/configuration.cpp

```diff
@@ -420,20 +422,60 @@ int hash_rules(const std::vector<std::string> &irbs, const int pid, std::string
 
 class make_copy {
 public:
-        make_copy(const std::vector<std::string>& _irbs, const int _pid) : irbs_(_irbs), pid_(_pid) {
-            for(auto const &irb : _irbs) {
-                boost::filesystem::copy_file(get_rule_base_path(irb), get_rule_base_path_copy(irb, _pid));
-            }
+  make_copy(const std::vector<std::string>& _irbs)
```

Comment:
Can we come up with a name for this class which better expresses its intent?

Reply:
Absolutely! Maybe `stored_rulebases` or `rulebase_files` or `rulebase_map`? Or something else?